### ISSUE TITLE
Failed to create AKS cluster, if 'DefaultResourceGroup-EUS' exists and is empty

### ISSUE THREAD

**Issue by mikkelhegn** [Azure/azure-cli#18625]
## Describe the bug

If an empty resource group named 'DefaultResourceGroup-EUS' exists, the below command fails.

**Command Name**
`az aks create`

**Errors:**
```
(ResourceNotFound) The Resource 'Microsoft.OperationalInsights/workspaces/DefaultWorkspace-c484c80e-0a6f-4470-86de-697ecee16984-EUS' under resource group 'DefaultResourceGroup-EUS' was not found. For more details please go to https://aka.ms/ARMResourceNotFoundFix

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- `az group create -n DefaultResourceGroup-EUS -l eastus`
- `az group create -n test -l eastus`
- `az aks create -g test -n test -l eastus --generate-ssh-keys --enable-addons monitoring`

## Expected Behavior

Either it works, or a better error message to know how to resolve.

## Environment Summary
```
Linux-5.4.72-microsoft-standard-WSL2-x86_64-with-debian-bullseye-sid
Python 3.6.10
Installer: DEB

azure-cli 2.24.2 *

Extensions:

```

**Comment by yonzhan** [comment]
route to service team

**Comment by mbifeld** [comment]
Hi @mikkelhegn, I realize this 4-year-old issue never got a meaningful response. Is this still an active issue?

**Comment by mikkelhegn** [comment]
TBH I have no idea :-)

**Comment by mbifeld** [comment]
Makes sense after 4 years! Sorry it didn't get a timely response. Closing this issue. Thank you!
